scss/utilities/_clearfix.scss

```diff
@@ -1,3 +1,11 @@
 .clearfix {
   @include clearfix();
 }
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-only($breakpoint) {
```

Comment:
I don't think we should use `media-breakpoint-up` here.

Look at the example:

in `xs` you have 1 col, in `sm` 2 cols, ... 

```html
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
  <div class="clearfix-sm"> <!-- Here we only want to clear on sm, not on md, lg & xl -->
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
  <div class="clearfix-md"> <!-- Here we only want to clear on md, not on lg & xl -->
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
  <div class="clearfix-sm clearfix-lg-up"> <!-- Here a clearfix-lg-up could be useful -->
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
  <div class="clearfix-sm clearfix-md">
<div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
```
We can use it as a extension, but than there also should be the `clearfix-xx-down`.